.github/workflows/playwright.yml

```diff
@@ -0,0 +1,32 @@
+name: Playwright Tests
+
+on:
+  push:
+    branches: [trunk]
+  pull_request:
+
+jobs:
+  PlaywrightE2ETests:
+    name: Playwright E2E tests
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        cache: 'npm' # Cache npm dependencies
+    - name: Install dependencies
+      run: npm ci
+    - name: Load wp-env
+      run: npm run env:start
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test --config=tests/e2e-pw/playwright.config.ts
+    - uses: actions/upload-artifact@v3
+      if: always()
```

Comment:
Are you referring to running the tests `npx playwright test [...]` to uploading the artifact?

If you're referring to running the tests, we could also use `npm run test:e2e-pw`, which is defined in https://github.com/woocommerce/woocommerce-blocks/blob/a043aee352ede55e7639d04a575eb0ae016b8c9f/package.json#L81

In case you're referring to the artifact, we could change `if: always()` to `if: failure()`, so that artifacts are only uploaded when a test fails.

Please let me know what you had in mind initially.

Reply:
I was referring to `if: always()` actually, but I was wondering why we need to specify the `if`? Can't we remove it, and won't it just upload anyway?

I think we can upload it every run, don't see an issue with that.